src/assets/css/oag.css

```diff
@@ -0,0 +1,85 @@
+.oag-theme header .brand img, .oag-theme header .brand object {
+	max-height: none;
+}
+
+.oag-theme header .oag-gb {
+	background: #1a303c;
+}
+
+.oag-theme .btn-primary{
+	background-color: #803C2C;
+	border-color: #26374a;
+	border-radius: unset;
+}
+
+.oag-theme .btn-primary:hover {
+	background-color: #1A303C;
+	border-color: #091c2d;
+}
+
+.oag-theme .alert-info {
+	border-color: #7F3D2D;
+	-o-border-image: linear-gradient(to bottom, #7F3D2D 16px, #7F3D2D 16px, transparent 16px, transparent 48px, #7F3D2D 48px, #7F3D2D 48px) 1 100%;
+	border-image: linear-gradient(to bottom, #7F3D2D 16px, #7F3D2D 16px, transparent 16px, transparent 48px, #7F3D2D 48px, #7F3D2D 48px) 1 100%;
+	color: #676C72;
+}
+
+.oag-theme .alert-info::before {
+	color: #7F3D2D;
+	content: "";
+}
+
+.oag-theme .text-primary {
+	color: #7F3D2D;
+}
+
+.oag-theme .label-primary{
+	background: #E3E0D3;
+	border-color: #908F84;
+	color: #4B5B62;
+}
+
+.oag-theme .well {
+	border: 2px solid #908F84;
+}
+
+.oag-theme .panel-default {
+	border: 1px solid transparent;
+	background-color: #F4F4F2 ;
+}
+
+.oag-theme .section-bg{
+	background-color: #F4F4F2;
+}
+
+.p-3 {
+	padding: 20px;
+}
+
+.pl-4 {
+	padding-left: 30px !important;
+}
+
+.pr-4 {
+	padding-right: 30px !important;
+}
```

Comment:
These lines are not necessary. They are already defined in GCWeb. See the newly added padding and margin utilities: https://wet-boew.github.io/GCWeb/common/spacing/spacing-en.html